force-app/main/default/classes/Homework.cls

```diff
@@ -81,15 +88,27 @@ public with sharing class Homework
 
         // convert the farenheight parameter to celsius and assign to
         // the celsius variable in as few statements as possible and
+
+        farenheight = (((celsius-32)*5)/9);
     }
 
     public Integer equals(String first, String second)
     {
         Integer result;
 
         // set result to 0 if first and second are equal
+       if (first==second){
+        result = 0 ;
+       }
+        
         // set result to 1 if first is greater than second 
+        if (first>second){
+            result = 1 ;
+           }
         // set result to -1 if first is less than second 
+        if (first<second){
+            result = -1 ;
+           }
 
         return result;
     } 
```

Comment:
This works, but could be more efficient. You'll execute the three `if` statements regardless of whether one matches. You could initialise `result` to 0, then check if it is greater than and use an else to check if less than:

```
Integer result=0;
if (first>second) 
{
    result=1;
}
else if (first<second)
{
    result=-1;
}
```